knimin/templates/pm_create_plate.html

```diff
@@ -0,0 +1,119 @@
+          var fields = {'plate_type': plateType, 'studies': studies,
+                        'plate_name': plateName};
+          $.each(fields, function(key, val) {
+            $('<input>').attr({type: "hidden", name: key, value: val}).appendTo(form);
+          });
+          form.appendTo('body').submit();
+        }
+        else {
+          // The plate name already exists
+          $('#plate-name-warning').html('* The given plate name already exists');
+          $('#plate-name-warning').show();
+        }
+      });
+    }
+  };
+
+  $(document).ready(function() {
+    // Code to dynamically add and remove studies input dropdowns
+    // Adapted from http://stackoverflow.com/q/24999006
+    $(document).on('click', '.btn-add', function(e) {
+      e.preventDefault();
+
+      var controlForm = $('.tag-controls');
+      var currentEntry = $(this).parents('.entry:first');
+
+      var newEntry = $("<div class='entry input-group'><select class='study-select'><option selected value='-1'>Choose a study</option></select><button class='btn btn-success btn-add' type='button'><span class='glyphicon glyphicon-plus'></span></button></div>");
+      add_study_options(newEntry.find('.study-select:first'));
+      newEntry.appendTo(controlForm);
+
+      $(".study-select").select2();
+      controlForm.find('.entry:not(:last) .btn-add')
+        .removeClass('btn-add').addClass('btn-remove')
+        .removeClass('btn-success').addClass('btn-info')
+        .html('<span class="glyphicon glyphicon-minus"></span>');
+    }).on('click', '.btn-remove', function(e) {
+        $(this).parents('.entry:first').remove();
+        e.preventDefault();
+        return false;
+    });
+
+    // Something
+    add_study_options($('#init-study-select'));
+
+    // Initialize the study autocompletion
+    $(".study-select").select2();
+  });
+</script>
+{% end %}
+{% block content %}
+
+<label><h3>Create new plate</h3></label>
+</br>
+<!-- Plate type selector -->
+<div class="container">
+<label>Plate type:</label>
+<select id='plate-type-select' class='form-control'>
+  {% for pt in plate_types %}
+    <option value="{{pt['id']}}">{{pt['name']}} ({{pt['notes']}})</option>
+  {% end %}
+</select>
+<!-- Studies selector -->
+<label>Studies being plated: <b id='studies-warning' class='warning' hidden>* Please choose at least one study</b></label>
+<div class="tag-controls">
+  <div class="entry input-group">
+    <select class="study-select" id='init-study-select'>
+        <option selected value='-1'>Choose a study</option>
+    </select>
+    <button class="btn btn-success btn-add" type="button"><span class="glyphicon glyphicon-plus"></span></button>
+  </div>
+</div>
+<label>Plate name: <b id='plate-name-warning' class='warning' hidden></b></label>
+<input class='form-control' id='plate-name'>
+</br>
+<div class="form-group">
+  <button onclick="submit_plate_creation();" class="btn button-sm btn-primary">Create plate</button>
+</div>
+</div>
```

Comment:
Nope, matches the `<div>` on line 95. I've updated the indentation to make it clear